src/App.js

```diff
@@ -1,27 +1,6 @@
-import './App.css';
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator/Calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```

Comment:
- [ ] Greate work creating the Calculator component however kindly make sure the calculator follows the provided design

|Your app|Provide design|
|-|-|
|![Screenshot_20230605_131649](https://github.com/10-menachi/calculator/assets/25227022/005aa0c9-a288-4bca-99ee-e39651995c74)|![Screenshot_20230605_132028](https://github.com/10-menachi/calculator/assets/25227022/61b26cc5-c3b3-4ceb-ae9d-3214905bc752)|